app/src/main/java/com/example/pokedex/repo/PokedexRepository.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.pokedex.repo
+
+import android.content.Context
+import com.example.pokedex.domain.PokemonList
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+
+interface PokedexRepository {
+    suspend fun getAllPokemon(): Call<PokemonList>
+}
+
+class PokedexRepositoryImpl(context: Context) : PokedexRepository {
```

Comment:
I originally didn't have it here. but then I put it here looking at the FTA, thinking it had to do with retrofit.

Then I realized it was probably because you were using localpref in that app and just forgot to remove it.

Reply:
Done.